frontend/pages/beta/model/new.tsx

```diff
@@ -82,83 +83,85 @@ export default function NewModel() {
 
   return (
     <Wrapper title='Create a new Model' page='upload'>
-      <Card sx={{ p: 4, maxWidth: 520, m: 'auto' }}>
-        <Typography
-          component='h1'
-          variant='h4'
-          sx={{ fontWeight: 'bold' }}
-          color='primary'
-          data-test='createModelPageTitle'
-        >
-          Create a new model
-        </Typography>
-        <Typography>A model repository contains all files, history and information related to a model.</Typography>
-        <Box component='form' sx={{ mt: 4 }} onSubmit={onSubmit}>
-          <Stack divider={<Divider orientation='vertical' flexItem />} spacing={2}>
-            <>
-              <Typography component='h2' variant='h6'>
-                Overview
-              </Typography>
-              <Box sx={{ width: '100%' }}>
-                <TeamAndModelSelector
-                  setTeamValue={setTeamName}
-                  teamValue={teamName}
-                  setModelValue={setModelName}
-                  modelValue={modelName}
-                />
-              </Box>
-              <Stack>
-                <FormControl>
-                  <Typography sx={{ fontWeight: 'bold' }}>
-                    Description <span style={{ color: theme.palette.primary.main }}>*</span>
-                  </Typography>
-                  <TextField
-                    data-test='modelDescription'
-                    onChange={(event) => setDescription(event.target.value)}
-                    value={description}
-                    size='small'
+      <Container maxWidth='sm'>
```

Comment:
Consider putting this `Container` inside `Wrapper` if it's something we're always going to wrap around the content.